aws/session/session_test.go

```diff
@@ -13,10 +13,10 @@ func TestCreateAWSSession(t *testing.T) {
 	// Set AWS_REGION to a made-up value to ensure no overlap with real regions
 	os.Setenv("AWS_REGION", "us-test-1")
 
-	sess := newSession(os.Getenv("AWS_PROFILE"), "")
+	_, sess := newSession(os.Getenv("AWS_PROFILE"), "")
```

Comment:
I think these are backwards? But also you can toss these because I've got a PR coming that'll include comprehensive tests for this. :)